### ISSUE TITLE
Simple learning rate schedule

### ISSUE THREAD

**Issue by gngdb** [BayesWatch/tf-variational-dropout#2]
Should train with SGD and two decay steps, in the traditional way.
